### DISCUSSION TITLE
在内网用centos部署了一个mosdns用不了

### DISCUSSION THREAD

**Discussion by langbinlee** [IrineSistiana/mosdns#688] [Q&A]
在内网用centos部署了一个mosdns用不了
设置网卡DNS为centos的IP后就无法解析网页，打不开网页。不知道怎么搞有没有熟悉的帮忙弄一下。红包答谢

**Comment by luweijun1992** [comment]
> 在内网用centos部署了一个mosdns用不了 设置网卡DNS为centos的IP后就无法解析网页，打不开网页。不知道怎么搞有没有熟悉的帮忙弄一下。红包答谢

可联系我wx交流看看
